src/services/moderation.service.ts

```diff
@@ -377,11 +451,10 @@ export class ModService {
     return await this._writeDataToFile(data);
   }
 
-  private async _getBanStatus(guild: Guild, id: string): Promise<string> {
+  private async _getBanStatus(guild: discord.Guild, id: string): Promise<string> {
```

Comment:
why not import {Guild,...} from discord ?

Reply:
There are so many imports the import statement is like 8 lines

Reply:
thats fine lol